app/controllers/course/stage/code-examples.ts

```diff
@@ -20,8 +20,9 @@ export default class CodeExamplesController extends Controller {
   rippleSpinnerImage = rippleSpinnerImage;
   @tracked order: 'recommended' | 'recent' | 'affiliated' = 'recommended';
   @tracked isLoading = true;
-  @tracked solutions: CommunityCourseStageSolutionModel[] = [];
   @tracked requestedLanguage: LanguageModel | null = null; // This shouldn't be state on the controller, see if we can move it to a query param or so?
+  @tracked solutions: CommunityCourseStageSolutionModel[] = [];
+  @tracked stageIncompleteModalWasDismissed = false;
```

Comment:
@libmartinito when this is stored on the controller, it'll persist between different stages. So if a user dismissed the modal on one stage, it'll remain dismissed on other stages too. 

Thoughts on moving this state to `CommunitySolutionList` instead?